build-system/global-configs/experiments-config.json

```diff
@@ -13,5 +13,11 @@
     "expiration_date_utc": "2021-01-01",
     "define_experiment_constant": "MOVE_FIXED_LAYER"
   },
-  "experimentC": {}
+  "experimentC": {
+    "name": "strict compilation",
```

Comment:
Perhaps use camel case to be consistent with experiments A and B?

Reply:
Do you know what the `name` impacts?  I don't have a great understanding of what happens with the experiment config 😬

Reply:
Pretty sure it's for documentation / logging within the release process. @danielrozenberg who may know more.